run_unit_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+LOGFILE=$(pwd)/unit_tests.log
+echo Using logfile $LOGFILE
+
+# Running npm test
+echo Running unit tests...
+npm test >> $LOGFILE
+
+UNIT_TEST_RESULT=$?
```

Comment:
Why not display the messages in the standard output? This will appear in Jenkins, right?